### PR TITLE
Feature: Add User-Page to token gallery

### PR DESCRIPTION
**Pull request by Ajadi-Abiola**
image
image

### CODE REVIEW

File: app/src/components/TokenCard.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const TokenCard = (props: ITokenCardProps): React.ReactElement => {
   return (
     <LinkBase target={`/tokens/${props.token.tokenId}`} isFullWidth={true}>
```
Comment (Ajadi-Abiola) [inline app/src/components/TokenCard.tsx:14]: Linkbase has prop isFullWidth= true, that is the reason it looks that way on desktop

Comment (krishan711) [reply app/src/components/TokenCard.tsx:14]: looks what way?

File: app/src/app.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ import { GlobalsProvider, IGlobals } from './globalsContext';
 import { PageDataProvider } from './PageDataContext';
 import { HomePage } from './pages/HomePage';
 import { getHomePageData } from './pages/HomePage/getHomePageData';
+import { UserPage } from './pages/UserPage/UserPage';
```
Comment (krishan711) [inline app/src/app.tsx:16]: since it has account in the url path, you should call this AccountPage instead of UserPage (change everywhere)

File: app/src/components/TokenCard.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ export const TokenCard = (props: ITokenCardProps): React.ReactElement => {
...
       <Stack direction={Direction.Vertical} shouldAddGutters={true} childAlignment={Alignment.Center} paddingBottom={PaddingSize.Default}>
-        <Box maxHeight='20em' minHeight='5em' shouldClipContent={true}>
+        <Box maxHeight='20em' minHeight='5em' maxWidth='20em' shouldClipContent={true}>
```
Comment (krishan711) [inline app/src/components/TokenCard.tsx:16]: why did you add this?

File: app/src/components/TokenDialog.tsx [hunk 4]
```diff
@@ -110,7 +110,7 @@ export const TokenDialog = (props: ITokenDialogProps): React.ReactElement => {
               ) : (
                 <Stack direction={Direction.Vertical} childAlignmentResponsive={{ base: Alignment.Center, medium: Alignment.Start }} paddingLeft={PaddingSize.Narrow}>
                   <Text variant='note'>Owner</Text>
-                  <LinkBase target={`https://nft.tokenhunt.io/accounts/${latestTransfer.toAddress}`}>
+                  <LinkBase target={`/accounts/${latestTransfer.toAddress}`}>
```
Comment (krishan711) [inline app/src/components/TokenDialog.tsx:113]: add a link in the navbar (or wherever we show the logged in user account)

File: app/src/components/TokenCard.tsx [hunk 5]
```diff
@@ -13,7 +13,7 @@ export const TokenCard = (props: ITokenCardProps): React.ReactElement => {
   return (
     <LinkBase target={`/tokens/${props.token.tokenId}`} isFullWidth={true}>
       <Stack direction={Direction.Vertical} shouldAddGutters={true} childAlignment={Alignment.Center} paddingBottom={PaddingSize.Default}>
-        <Box maxHeight='20em' minHeight='5em' shouldClipContent={true}>
+        <Box maxHeight='20em' minHeight='5em'shouldClipContent={true}>
```
Comment (krishan711) [inline app/src/components/TokenCard.tsx:16]: undo

**Review by krishan711 — APPROVED** [review]
